src/components/Organisms/TodoCard/index.jsx

```diff
@@ -0,0 +1,57 @@
+import React, { useState } from "react";
+import AddTaskButton from "../../Atoms/AddTaskButton";
+import Task from "../../Molecules/Task";
+import styled from "styled-components";
+import COLOR from "../../../variables/color";
+
+const TodoCard = () => {
+  const [task, setNewTask] = useState([]);
+  const pushAddbutton = () => {
+    setNewTask([...task, { name: "", state: "TODO" }]);
+  };
+  const editingTask = task.map(({ name, state }, index) => {
+    if (state == "TODO") {
```

Comment:
```suggestion
    if (state === "TODO") {
```
基本的に厳密比較を使うようにしてください。